packages/indy-vdr/src/index.ts

```diff
@@ -6,3 +6,8 @@ try {
 } catch (error) {
   throw new Error('Error registering nodejs bindings for Indy VDR')
```

Comment:
I think it will be better to remove this try-catch here and put it in Module's register() method as we did for askar and anoncreds-rs. Note that here you are checking the presence of nodejs wrapper, while in the other cases we are trying to import react-native before throwing the error.

Reply:
Agreed, not sure why this is happening here.